internal/generate/testdata/clusterserviceversions/output/memcached-operator.clusterserviceversion.yaml

```diff
@@ -8,10 +8,10 @@ metadata:
           "apiVersion": "cache.example.com/v1alpha1",
```

Comment:
These testdata changes can be refactored into another PR, they're not directly related to the bugfix.